core-web/package.json

```diff
@@ -196,6 +198,7 @@
         "@types/webpack": "4.41.21",
         "@typescript-eslint/eslint-plugin": "6.21.0",
         "@typescript-eslint/parser": "6.21.0",
+        "@vitest/ui": "^1.0.4",
```

Comment:
Don't go here, stick to jest, keep it consistent.

Reply:
Instaled by vite, removed.